toml.md

```diff
@@ -104,27 +104,39 @@ Keys
 
 A key may be either bare, quoted, or dotted.
 
-**Bare keys** may only contain ASCII letters, ASCII digits, underscores, and
-dashes (`A-Za-z0-9_-`). Note that bare keys are allowed to be composed of only
-ASCII digits, e.g. `1234`, but are always interpreted as strings.
+**Bare keys** may contain any letter-like Unicode character from any Unicode script,
```

Comment:
Should we take this opportunity to audit the spec for "character" vs "code point"? My personal feeling is that we should avoid the word "character" when talking about Unicode, and prefer the more technically-correct "code point", since "character" tends to mean lots of different things to different people (it even means different things in Unicode itself!).

Reply:
@marzer, the word "character" is used 21 times in the document. I agree that "code-point" is better, at least in most cases. In Unicode, a character can be, say `é`, but it can be either one or two code-points. For characters in some other langs, like Arabic, there can be many more codepoints that comprise a single character.

Reply:
>The only argument against using "code-point" is that a large swatch of people won't know what that means...

Yeah, though implementers absolutely must know what it means, so for contexts where the prose is largely implementer-focused (e.g. listing acceptable code point ranges) I'd say that's not too much of an issue. 

I don't consider myself anything close to a unicode expert, though, so I don't really have any suggestions about wording changes. Happy to defer to your judgement here.

Reply:
My 2¢ is that this seems a bit of a "technically correct" vs. "perhaps not entirely technically correct, but everyone knows what you mean and it reads a bit better"-discussions. AFAIK no one has ever been confused by "character", so I think it's fine to just keep it.

Reply:
@arp242 I'm content with that reasoning too. If "character" is unambiguous in the general case, it's probably fine. Just thought I'd flag it as specs tend to be more precise with Unicode terminology.

Reply:
Per the comment below by @marzer, we'll take this discussion to a new issue, if necessary, and leave the current status quo w.r.t. to the use of the word "character" as-is.